apps/ledger-live-mobile/ios/ledgerlivemobile/ledgerlivemobile.entitlements

```diff
@@ -2,12 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
 	<key>com.apple.developer.associated-domains</key>
 	<array>
 		<string>applinks:r354.adj.st</string>
+		<string>applinks:ledger.go.link</string>
+		<string>applinks:a4wc.adj.st</string>
+		<string>applinks:ledger-debug.go.link</string>
+		<string>applinks:fvsc.adj.st</string>
+		<string>applinks:ledger-staging.go.link</string>
```

Comment:
Added all supported universal links domains in entitlements file, at `com.apple.developer.associated-domains` key